src/user-statuses/userStatusTextSuggestions.js

```diff
@@ -1,11 +1,19 @@
 /* @flow strict-local */
 
+type Item = [
+  $ReadOnly<{| emoji_name: string, emoji_code: string, reaction_type: 'unicode_emoji' |}>,
+  string,
+];
+
 // This is a separate file due to Prettier having issues with formatting unicode
 // Using `prettier-ignore` does not help.
-export default [
-  '📅 In a meeting',
-  '🚌 Commuting',
-  '🤒 Out sick',
-  '🌴 Vacationing',
-  '🏠 Working remotely',
-];
+const items: $ReadOnlyArray<Item> = [
+  [{ emoji_name: 'working_on_it', emoji_code: '1f6e0' }, 'Busy'],
```

Comment:
Let's get these codes from `unicodeCodeByName`. That way there isn't a risk of the name and code disagreeing with each other, or getting out of sync if our names change.